src/Riok.Mapperly/Emit/SyntaxFactoryHelper.cs

```diff
@@ -350,4 +352,10 @@ private static IEnumerable<SyntaxNodeOrToken> Join(SyntaxToken sep, bool insertT
             yield return sep;
         }
     }
+
+    public static IdentifierNameSyntax FullyQualifiedIdentifier(ITypeSymbol typeSymbol)
+    => IdentifierName(FullyQualifiedIdentifierName(typeSymbol));
+
+    public static string FullyQualifiedIdentifierName(ITypeSymbol typeSymbol)
+    => typeSymbol.ToDisplayString(_fullyQualifiedNullableFormat);
```

Comment:
Nit: Align indention of the expression to the same level as all other expressions in this class and probably move these two methods after `NonNullableIdentifier` since they belong together.